src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -50,6 +72,18 @@ TableToolbar.propTypes = {
    * Function used to export a file in Table. If undefined, export button is not displayed
    */
   onExport: PropTypes.func,
+  /*
+   * Function used to add a row in the Table. If it's not defined, he not not display the add row button
```

Comment:
better phrasing: If it's not defined, **the button won't be displayed**

Reply:
DONE